moqt-server/src/modules/moqt_client.rs

```diff
@@ -44,4 +48,18 @@ impl MOQTClient {
 
         Ok(())
     }
+
+    pub fn senders(&self) -> Arc<Senders> {
+        self.senders.clone().unwrap()
+    }
+
+    #[cfg(test)]
+    pub fn new_without_senders(id: usize) -> Self {
```

Comment:
I think it can be deleted.
we can use mock or dummy senders.

Reply:
I have fixed.